ansible/roles/stack-sunbird/templates/sunbird_notification-service.env

```diff
@@ -1,3 +1,15 @@
+{% if (cassandra_cluster_size | int) > 1 %}
+sunbird_cassandra_host={{groups['cassandra']|join(',')}}
+sunbird_cassandra_port=9042,9042,9042
+sunbird_cassandra_consistency_level=quorum
+{% else %}
+sunbird_cassandra_host={{ groups['cassandra'][0] }}
+sunbird_cassandra_port=9042
+sunbird_cassandra_consistency_level=quorum
+{% endif %}
+sunbird_cassandra_keyspace=sunbird_notifications
```

Comment:
can you update the variable name to sunbird_cassandra_notification_keyspace @sknirmalkar89